[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:12,883 ERROR [DiscoveryClient-HeartbeatExecutor-0][RedirectingEurekaHttpClient.java:83] - Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:53000/eureka/}
com.sun.jersey.api.client.ClientHandlerException: java.net.ConnectException: Connection refused: connect
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:187)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:123)
	at com.netflix.discovery.EurekaIdentityHeaderFilter.handle(EurekaIdentityHeaderFilter.java:27)
	at com.sun.jersey.api.client.Client.handle(Client.java:652)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:682)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.put(WebResource.java:529)
	at com.netflix.discovery.shared.transport.jersey.AbstractJerseyEurekaHttpClient.sendHeartBeat(AbstractJerseyEurekaHttpClient.java:102)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$3.execute(EurekaHttpClientDecorator.java:92)
	at com.netflix.discovery.shared.transport.decorator.MetricsCollectingEurekaHttpClient.execute(MetricsCollectingEurekaHttpClient.java:73)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.sendHeartBeat(EurekaHttpClientDecorator.java:89)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$3.execute(EurekaHttpClientDecorator.java:92)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.executeOnNewServer(RedirectingEurekaHttpClient.java:118)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.execute(RedirectingEurekaHttpClient.java:79)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.sendHeartBeat(EurekaHttpClientDecorator.java:89)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$3.execute(EurekaHttpClientDecorator.java:92)
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:120)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.sendHeartBeat(EurekaHttpClientDecorator.java:89)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$3.execute(EurekaHttpClientDecorator.java:92)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.sendHeartBeat(EurekaHttpClientDecorator.java:89)
	at com.netflix.discovery.DiscoveryClient.renew(DiscoveryClient.java:846)
	at com.netflix.discovery.DiscoveryClient$HeartbeatThread.run(DiscoveryClient.java:1405)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:121)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:180)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:134)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:605)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:440)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:118)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:173)
	... 27 more
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:12,886 WARN [DiscoveryClient-HeartbeatExecutor-0][RetryableEurekaHttpClient.java:130] - Request execution failed with message: java.net.ConnectException: Connection refused: connect
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:12,887 ERROR [DiscoveryClient-HeartbeatExecutor-0][DiscoveryClient.java:860] - DiscoveryClient_GATEWAY-SERVER/gateway-server:10.1.21.102:53010 - was unable to send heartbeat!
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.sendHeartBeat(EurekaHttpClientDecorator.java:89)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$3.execute(EurekaHttpClientDecorator.java:92)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.sendHeartBeat(EurekaHttpClientDecorator.java:89)
	at com.netflix.discovery.DiscoveryClient.renew(DiscoveryClient.java:846)
	at com.netflix.discovery.DiscoveryClient$HeartbeatThread.run(DiscoveryClient.java:1405)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:12,911 ERROR [DiscoveryClient-CacheRefreshExecutor-0][RedirectingEurekaHttpClient.java:83] - Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:53000/eureka/}
com.sun.jersey.api.client.ClientHandlerException: java.net.ConnectException: Connection refused: connect
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:187)
	at com.sun.jersey.api.client.filter.GZIPContentEncodingFilter.handle(GZIPContentEncodingFilter.java:123)
	at com.netflix.discovery.EurekaIdentityHeaderFilter.handle(EurekaIdentityHeaderFilter.java:27)
	at com.sun.jersey.api.client.Client.handle(Client.java:652)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:682)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.get(WebResource.java:509)
	at com.netflix.discovery.shared.transport.jersey.AbstractJerseyEurekaHttpClient.getApplicationsInternal(AbstractJerseyEurekaHttpClient.java:194)
	at com.netflix.discovery.shared.transport.jersey.AbstractJerseyEurekaHttpClient.getDelta(AbstractJerseyEurekaHttpClient.java:170)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$7.execute(EurekaHttpClientDecorator.java:152)
	at com.netflix.discovery.shared.transport.decorator.MetricsCollectingEurekaHttpClient.execute(MetricsCollectingEurekaHttpClient.java:73)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.getDelta(EurekaHttpClientDecorator.java:149)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$7.execute(EurekaHttpClientDecorator.java:152)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.executeOnNewServer(RedirectingEurekaHttpClient.java:118)
	at com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient.execute(RedirectingEurekaHttpClient.java:79)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.getDelta(EurekaHttpClientDecorator.java:149)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$7.execute(EurekaHttpClientDecorator.java:152)
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:120)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.getDelta(EurekaHttpClientDecorator.java:149)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$7.execute(EurekaHttpClientDecorator.java:152)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.getDelta(EurekaHttpClientDecorator.java:149)
	at com.netflix.discovery.DiscoveryClient.getAndUpdateDelta(DiscoveryClient.java:1085)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:967)
	at com.netflix.discovery.DiscoveryClient.refreshRegistry(DiscoveryClient.java:1479)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1446)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:121)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:180)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:134)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:605)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:440)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:118)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:173)
	... 30 more
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:12,914 WARN [DiscoveryClient-CacheRefreshExecutor-0][RetryableEurekaHttpClient.java:130] - Request execution failed with message: java.net.ConnectException: Connection refused: connect
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:12,915 ERROR [DiscoveryClient-CacheRefreshExecutor-0][DiscoveryClient.java:972] - DiscoveryClient_GATEWAY-SERVER/gateway-server:10.1.21.102:53010 - was unable to refresh its cache! status = Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.getDelta(EurekaHttpClientDecorator.java:149)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$7.execute(EurekaHttpClientDecorator.java:152)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.getDelta(EurekaHttpClientDecorator.java:149)
	at com.netflix.discovery.DiscoveryClient.getAndUpdateDelta(DiscoveryClient.java:1085)
	at com.netflix.discovery.DiscoveryClient.fetchRegistry(DiscoveryClient.java:967)
	at com.netflix.discovery.DiscoveryClient.refreshRegistry(DiscoveryClient.java:1479)
	at com.netflix.discovery.DiscoveryClient$CacheRefreshThread.run(DiscoveryClient.java:1446)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:14,094 INFO [NettyClientSelector_1][Slf4jLoggerFactory.java:95] - closeChannel: close the connection to remote address[] result: true
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:14,203 INFO [NettyClientSelector_1][Slf4jLoggerFactory.java:95] - closeChannel: close the connection to remote address[] result: true
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:16,102 INFO [NettyClientSelector_1][Slf4jLoggerFactory.java:95] - closeChannel: close the connection to remote address[] result: true
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:16,806 INFO [hmilyTransaction-hmily-transaction-self-recovery-2][LogUtil.java:87] - self recovery execute delayTime:60
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,202 INFO [hmilyTransaction-hmily-transaction-self-recovery-10][LogUtil.java:87] - self recovery execute delayTime:60
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,304 INFO [hmilyTransaction-hmily-transaction-self-recovery-20][LogUtil.java:87] - self recovery execute delayTime:60
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,306 INFO [hmilyTransaction-hmily-transaction-self-recovery-30][LogUtil.java:87] - self recovery execute delayTime:60
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,797 WARN [Apollo-RemoteConfigLongPollService-1][RemoteConfigLongPollService.java:193] - Long polling failed, will retry in 120 seconds. appId: depository-agent-service, cluster: DEFAULT, namespaces: micro_service.spring-boot-redis+micro_service.spring-boot-http+micro_service.spring-cloud-feign+micro_service.mybatis-plus+application+micro_service.spring-rocketmq+micro_service.spring-ribbon+micro_service.spring-boot-druid+micro_service.spring-eureka+micro_service.spring-hystrix, long polling url: http://10.1.21.102:8080/notifications/v2?cluster=DEFAULT&appId=depository-agent-service&ip=10.1.21.102&notifications=%5B%7B%22namespaceName%22%3A%22micro_service.spring-cloud-feign%22%2C%22notificationId%22%3A403%7D%2C%7B%22namespaceName%22%3A%22micro_service.mybatis-plus%22%2C%22notificationId%22%3A461%7D%2C%7B%22namespaceName%22%3A%22application%22%2C%22notificationId%22%3A467%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-rocketmq%22%2C%22notificationId%22%3A354%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-ribbon%22%2C%22notificationId%22%3A351%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-druid%22%2C%22notificationId%22%3A477%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-redis%22%2C%22notificationId%22%3A10%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-eureka%22%2C%22notificationId%22%3A431%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-hystrix%22%2C%22notificationId%22%3A350%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-http%22%2C%22notificationId%22%3A462%7D%5D, reason: Could not complete get operation [Cause: connect timed out]
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,800 WARN [Apollo-RemoteConfigLongPollService-1][RemoteConfigLongPollService.java:193] - Long polling failed, will retry in 120 seconds. appId: transaction-service, cluster: DEFAULT, namespaces: micro_service.spring-boot-http+micro_service.spring-cloud-feign+micro_service.mybatis-plus+application+micro_service.spring-ribbon+micro_service.spring-eureka+micro_service.spring-hystrix, long polling url: http://10.1.21.102:8080/notifications/v2?cluster=DEFAULT&appId=transaction-service&ip=10.1.21.102&notifications=%5B%7B%22namespaceName%22%3A%22micro_service.spring-cloud-feign%22%2C%22notificationId%22%3A403%7D%2C%7B%22namespaceName%22%3A%22micro_service.mybatis-plus%22%2C%22notificationId%22%3A471%7D%2C%7B%22namespaceName%22%3A%22application%22%2C%22notificationId%22%3A482%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-ribbon%22%2C%22notificationId%22%3A287%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-eureka%22%2C%22notificationId%22%3A431%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-hystrix%22%2C%22notificationId%22%3A286%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-http%22%2C%22notificationId%22%3A469%7D%5D, reason: Could not complete get operation [Cause: connect timed out]
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,812 WARN [Apollo-RemoteConfigLongPollService-1][RemoteConfigLongPollService.java:193] - Long polling failed, will retry in 120 seconds. appId: account-service, cluster: DEFAULT, namespaces: micro_service.spring-boot-redis+micro_service.spring-boot-http+micro_service.spring-cloud-hmily+micro_service.mybatis-plus+application+micro_service.spring-rocketmq+micro_service.spring-ribbon+micro_service.spring-boot-druid+micro_service.spring-eureka+micro_service.spring-hystrix, long polling url: http://10.1.21.102:8080/notifications/v2?cluster=DEFAULT&appId=account-service&ip=10.1.21.102&notifications=%5B%7B%22namespaceName%22%3A%22micro_service.spring-cloud-hmily%22%2C%22notificationId%22%3A478%7D%2C%7B%22namespaceName%22%3A%22micro_service.mybatis-plus%22%2C%22notificationId%22%3A446%7D%2C%7B%22namespaceName%22%3A%22application%22%2C%22notificationId%22%3A475%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-rocketmq%22%2C%22notificationId%22%3A364%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-ribbon%22%2C%22notificationId%22%3A360%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-druid%22%2C%22notificationId%22%3A477%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-redis%22%2C%22notificationId%22%3A362%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-eureka%22%2C%22notificationId%22%3A431%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-hystrix%22%2C%22notificationId%22%3A359%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-http%22%2C%22notificationId%22%3A436%7D%5D, reason: Could not complete get operation [Cause: connect timed out]
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,817 WARN [Apollo-RemoteConfigLongPollService-1][RemoteConfigLongPollService.java:193] - Long polling failed, will retry in 120 seconds. appId: consumer-service, cluster: DEFAULT, namespaces: micro_service.spring-boot-redis+micro_service.spring-boot-http+micro_service.spring-cloud-hmily+micro_service.mybatis-plus+application+micro_service.spring-rocketmq+micro_service.spring-ribbon+micro_service.spring-boot-druid+micro_service.spring-eureka+micro_service.spring-hystrix, long polling url: http://10.1.21.102:8080/notifications/v2?cluster=DEFAULT&appId=consumer-service&ip=10.1.21.102&notifications=%5B%7B%22namespaceName%22%3A%22micro_service.spring-cloud-hmily%22%2C%22notificationId%22%3A478%7D%2C%7B%22namespaceName%22%3A%22micro_service.mybatis-plus%22%2C%22notificationId%22%3A446%7D%2C%7B%22namespaceName%22%3A%22application%22%2C%22notificationId%22%3A441%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-rocketmq%22%2C%22notificationId%22%3A464%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-ribbon%22%2C%22notificationId%22%3A6%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-druid%22%2C%22notificationId%22%3A480%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-redis%22%2C%22notificationId%22%3A10%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-eureka%22%2C%22notificationId%22%3A431%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-hystrix%22%2C%22notificationId%22%3A5%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-http%22%2C%22notificationId%22%3A399%7D%5D, reason: Could not complete get operation [Cause: connect timed out]
[${project.name}][${env:SERVER_PORT}] 2021-09-06 17:00:18,822 WARN [Apollo-RemoteConfigLongPollService-1][RemoteConfigLongPollService.java:193] - Long polling failed, will retry in 120 seconds. appId: gateway-server, cluster: DEFAULT, namespaces: micro_service.spring-boot-redis+micro_service.spring-boot-http+application+micro_service.spring-ribbon+micro_service.spring-eureka+micro_service.spring-hystrix, long polling url: http://10.1.21.102:8080/notifications/v2?cluster=DEFAULT&appId=gateway-server&ip=10.1.21.102&notifications=%5B%7B%22namespaceName%22%3A%22application%22%2C%22notificationId%22%3A473%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-ribbon%22%2C%22notificationId%22%3A187%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-redis%22%2C%22notificationId%22%3A10%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-eureka%22%2C%22notificationId%22%3A431%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-hystrix%22%2C%22notificationId%22%3A5%7D%2C%7B%22namespaceName%22%3A%22micro_service.spring-boot-http%22%2C%22notificationId%22%3A399%7D%5D, reason: Could not complete get operation [Cause: connect timed out]
